### PR TITLE
Closes #69 Add m4a media type

### LINKED ISSUE #69: add .m4a file type to media-types.js
Audio files ending in `m4a` are not processed correctly because this extension isn't defined in `lib/markout/media-types.js`.

It should be added so that these files get processed automatically.

### PR DESCRIPTION
**Pull request by arolos**

### CODE REVIEW

**Review by kphunter — DISMISSED** [review]
Can you just double-check that `video/mp4` is the best way to represent `.m4a` file types? Thx.

**Comment by arolos** [comment]
@kphunter To my understanding **m4a (audio without video)** is the same format as **mp4 (audio and video)**. Mozilla haven't mention mp4 in their [page](https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats) because the table still in the old format, but I found a reference in [wiki](https://en.wikipedia.org/wiki/HTML5_audio#Supported_audio_coding_formats).

**Comment by kphunter** [comment]
@Arv91 Thanks for clarifying!

File: lib/markout/media-types.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ module.exports = function (media, src) {
 		case ".wav":
 			type = "wav";
 			break;
-		case ".mp4":
+        case ".mp4":
```
Comment (MCTaylor17) [inline lib/markout/media-types.js:26]: My OCD 😨

Comment (arolos) [reply lib/markout/media-types.js:26]: Ah.. the space vs tab problem

**Pull request merged**